crowbar_framework/app/models/deployer_service.rb

```diff
@@ -151,35 +175,8 @@ def transition(inst, name, state)
         end
       end
 
-      # Let it fly to the provisioner. Reload to get the address.
       node = NodeObject.find_node_by_name node.name
-      node.crowbar["crowbar"]["usedhcp"] = true
-
-      role = RoleObject.find_role_by_name "deployer-config-#{inst}"
-      unless role.default_attributes["deployer"]["use_allocate"] and !node.admin?
-        @logger.debug("Automatically allocating node #{node.name}")
-        node.allocate!
-      end
-
-      # save this on the node after it's been refreshed with the network info.
-      node.crowbar["crowbar"]["boot_ip_hex"] = boot_ip_hex  if boot_ip_hex
-      node.save
-
-      @logger.debug("Deployer transition: leaving discovered for #{name} EOF")
-      return [200, { name: name }]
-    end
-
-    #
-    # Once we have been allocated, we will fly through here and we will setup the raid/bios info
-    #
-    if state == "hardware-installing"
-      # build a list of current and pending roles to check against
-      roles = []
-      node.crowbar["crowbar"]["pending"].each do |k,v|
-        roles << v
-      end unless node.crowbar["crowbar"]["pending"].nil?
-      roles << node.run_list_to_roles
-      roles.flatten!
+      save_it = true
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `save_it`. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)